bamazonBasic.js

```diff
@@ -0,0 +1,92 @@
+var CRUD = require("./crud.js");
+var inquirer = require("inquirer");
+var fs = require("fs");
+require("console.table");
+
+
+var Bamazon = function(database, password) {
```

Comment:
since the app is called bamazon I would rename this function to something that is relevant to what the function is doing, because it is setting up your crud user interface i would name it something like "bamazonInterface" or "userInterface".